tensorflow/lite/micro/kernels/xtensa/conv_vision.cc

```diff
@@ -0,0 +1,140 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
```

Comment:
Please fix formatting issues with the copyright info https://github.com/tensorflow/tflite-micro/runs/4655335928?check_suite_focus=true

Reply:
I have pushed a commit with these fixes.